upload/views/uploads.py

```diff
@@ -56,6 +57,18 @@ def create_upload(
         else None
     )
     archive_service = ArchiveService(repository)
+    # only Shelter requests are allowed to set their own `storage_path`
+    serializer.validated_data["external_id"] = uuid.uuid4()
```

Comment:
We should be able to directly do `uploadid=uuid.uuid4()`, otherwise we'd be creating this external ID and not using it most of the time. 

If we need to store this external_id in the DB, then I'd just move this line bw 62 and 63 and the rest would be the same

Reply:
Good point. I moved it down. I assume we need it because of how it was being done before.